packages/core/src/fhirpath/functions.ts

```diff
@@ -1482,11 +1489,10 @@ export const functions: Record<string, FhirPathFunction> = {
    * See: https://hl7.org/fhirpath/#tracename-string-projection-expression-collection
    * @param context The evaluation context.
    * @param input The input collection.
-   * @param nameAtom The log name.
+   * @param _nameAtom The log name.
```

Comment:
linter error

Reply:
The `console.log()` was a necessary part of the trace function; it shouldn't be removed